about.md

```diff
@@ -1,30 +1,101 @@
+
+- **Funding**: There is no funding model for GeoBlacklight, and most development comes through volunteered or assigned time from contributing organizations. Some organizations receive grants or funding to build their GeoBlacklight applications or contribute to the community, and there are several vendors and independent contractors that have contributed through contracted work.
+
+- **Documentation**: The technical documentation for the project resides in the [project wiki](https://github.com/geoblacklight/geoblacklight/wiki) and within our [tutorials](https://geoblacklight.org/tutorials.html).
+
+**************************
+
+Application Features
+--------------------
+
+- **Technical Stack**: GeoBlacklight is a Ruby on Rails engine designed as a plugin for the popular open source discovery framework, Blacklight. GeoBlacklight has direct software dependencies to Blacklight and Ruby on Rails. It also requires searchable metadata through the Apache Solr project. For production uses, GeoBlacklight installations often require a SQL database such as MySQL, MariaDB, or PostgreSQL. 
+
+- **External service integration**: GeoBlacklight provides a discovery layer for content that is oftentimes hosted elsewhere. Rather than try to provide a repository to the data itself and navigate the complexities of different adopters existing systems, GeoBlacklight relies on providing integration for existing datastores and web services. GeoBlacklight does this through providing useful preview, download, and exports of open standards-based services, including Web Mapping Services (WMS), Web Feature Services (WFS), ArcGIS Rest API, and International Image Interoperability Framework (IIIF). There is also support for externally referenced metadata viewing and file download support.
+
+- **Metadata**: The GeoBlacklight Metadata Schema has been designed to privilege discovery use cases. Text search, faceted search & refinement, and spatial search & relevancy are among the primary features that the schema enables. [Visit the Metadata Wiki for more information and full documentation.](https://github.com/geoblacklight/geoblacklight/wiki/GeoBlacklight-Metadata)
+
+	- based on Dublin Core, with custom elements added for spatial values
+
+	- designed for discovery - to help users find items
+
+	- _not_ designed for complete technical documentation, such as a GIS dataset's processing history
+
+	- includes elements for external links, such as downloads, web services, or supplemental metadata
+
+	- records are formatted as JSON files
+
+**************************
+
+GeoBlacklight Technical Values
+------------------------------
+
+- Our core focus is **geospatial discovery**. This focus initially was limited to discretely catalogued data objects, but has expanded over time to include a wider range of information sources.
+
+- We emphasize **end-user experience**, including inclusivity and accessibility in design features whenever possible.
```

Comment:
I would put a comma after features in this sentence

Reply:
Good note. I made an edit that just removes the clause "whenever possible" as it doesn't seem (to me) to be needed here.